src/components/pager/index.js

```diff
@@ -0,0 +1,367 @@
+import { vlElement, define } from "../../utils/core";
+import styles from "./styles.scss";
+
+/**
+ * Pager changed event
+ * @event VlPager#change
```

Comment:
Dit is een belangrijk event dat niet in uw docs vermeld staan. https://uig-webcomponents.omgeving.vlaanderen.be/?path=/story/custom-elements-vl-progress-bar--default hier heb ik bv ook een custom event in de docs gestoken.

Reply:
Fixed.